### ISSUE TITLE
9.4.1.2/2.2 The "Weather" menu (see screenhot) has been implemented using the aria-haspopup="menu" attribute, which informs users that a menu is being opened. However, even though the menu items can be navigated using arrow keys, the corresponding ARIA roles role="menu" and role="menuitem" are completely missing.

### ISSUE THREAD

**Issue by szaimen** [nextcloud/server#42228]
https://report.bitvtest.de/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-bfbf07e5c3-v2-n2

9.4.1.2/2.2 The "Weather" menu (see screenhot) has been implemented using the aria-haspopup="menu" attribute, which informs users that a menu is being opened. However, even though the menu items can be navigated using arrow keys, the corresponding ARIA roles role="menu" and role="menuitem" are completely missing.

![image](https://github.com/nextcloud/server/assets/42591237/dc3763c3-ec29-46ae-9dff-0702a5c092b0)

**Comment by JuliaKirschenheuter** [comment]
duplicate of https://github.com/nextcloud/server/issues/41889
